source/atlas-cli-getting-started.txt

```diff
@@ -0,0 +1,135 @@
+
+Follow These Steps
+------------------
+
+Complete the following procedure to get started with |service|.
+
+.. procedure::
+
+   .. step:: Run the ``atlas setup`` command in the terminal.
+
+      .. code-block:: sh
+      
+         atlas setup
+         
+      After you run the command, enter :guilabel:`Y` to open the default browser. A browser window displays the :guilabel:`Create Your Account`
+      screen.
+
+      If you want to log into an existing |service| account, click
+      :guilabel:`Log in now` and log in.
+
+      If you're already logged into an existing |service| account,
+      proceed to step 3.
+
+   .. step:: Sign up and verify your account.
+
+      Enter your account information and click :guilabel:`Sign Up`.
+      Follow the prompts to verify your email or register using
+      third-party authentication.
+
+   .. step:: Verify your {+atlas-cli+} session.
+
+      When you reach the :guilabel:`Activation` screen, copy the
+      verification code from the {+atlas-cli+} and paste it into the
+      browser. Then, click :guilabel:`Confirm Authorization` and return
+      to the terminal window.
+
+   .. step:: Accept the default {+cluster+} creation settings.
+
+      After you verify your {+atlas-cli+} session, ``atlas setup``
+      creates an ``M0`` cluster. ``M0`` clusters have some operational
+      :ref:`limitations <atlas-free-tier>`.
+
+      If you logged into an existing account and had existing
+      organizations and projects, ``atlas setup`` prompts you to select
+      a default organization and default project. Select a default
+      organization and project and press
+      :guilabel:`Enter`.
+      
+      When the {+atlas-cli+} prompts ``Do you want to set up``
+      ``your first free database in Atlas with default``
+      ``settings? It's free forever!``, enter :guilabel:`Y` to create your {+cluster+} with the default settings.
+            
+      The command creates a sample ``M0`` shared-tier cluster with the 
+      following default settings:
+ 
+      - Cluster name: ``Cluster<number>``
+      - Cloud provider and region: ``AWS - US_EAST_1``
+      - Database Username: ``Cluster<number>``
+      - Database User Password: ``abcdef12345``
+      - Load Sample Data: ``Yes``
+      - Allow connection from IP: ``<YourIPAddress>``
+  
+      .. io-code-block::
+         
+         .. input::
+            :language: sh
+            
+            Do you want to set up your first free database in
+            Atlas with default settings? It's free forever! Y
+
+         .. output::
+            :language: sh
+
+            We are deploying Cluster9876543...
+
+            Please store your database authentication access details in a secure location.
+            Database User Username: Cluster9876543
+            Database User Password: abcdef12345
+
+            Connection string: mongosh -u Cluster-9876543 -p abcdef12345 mongod
```

Comment:
I'm checking if the connection string will be showing here. At the moment I believe it is not, we were planning to add this but I think there is a technical blocker. Get back in ~20 min hopefully.

Reply:
For the time being, we can show the same as we show in quickstart, the "Connection string" sentence in the end of the snippet